### ISSUE TITLE
pretrain

### ISSUE THREAD

**Issue by 1190202002** [myavartanoo/CNC-Net_PyTorch#2]
Hello, I want to do model pretraining, but I found that you don't provide pretraining code. I wrote a pre-training process myself, but the loss function is difficult to decrease, and the training effect is worse than zero-shot. I think it may be that there is something wrong with my code. Can you provide pre-trained code or trained weight files? Thanks
